Student/StudentUITests/Calendar/CalendarTests.swift

```diff
@@ -29,93 +29,194 @@ enum CalendarElements {
     static func text(containing text: String) -> Element {
         return app.find(labelContaining: text)
     }
+
+    static var refreshButton: Element {
+        return app.find(id: "CalendarMonthViewController.refreshButton")
+    }
+
+    static func dayLabel(_ date: Date) -> Element {
+        let components = Calendar.current.dateComponents([.year, .month, .day], from: date)
+        return app.find(id: "\(components.year!)-\(components.month!)-\(components.day!)-label")
+    }
+
+    static func dayEventIndicator(_ date: Date) -> Element {
+        let components = Calendar.current.dateComponents([.year, .month, .day], from: date)
+        return app.find(id: "\(components.year!)-\(components.month!)-\(components.day!)-eventIndicator")
+    }
 }
 
 class CalendarTests: StudentUITestCase {
     lazy var today = Date(fromISOString: "2007-06-04T12:00:00Z")!
     lazy var tomorrow = today.addDays(1)
 
-    lazy var event1: APICalendarEvent = APICalendarEvent.make(start_at: today, end_at: today)
-    lazy var event2: APICalendarEvent = APICalendarEvent.make(id: "2", title: "event 2", start_at: tomorrow, end_at: tomorrow)
+    lazy var jan1 = DateComponents(calendar: .current, timeZone: .current, year: 2019, month: 1, day: 1).date!
+    lazy var jan2 = jan1.addDays(1)
+    lazy var jan3 = jan2.addDays(1)
+    lazy var jan4 = jan3.addDays(1)
```

Comment:
With 5 different jan dates, (`jan4.addDays(1)` below) this might by past the point to abstract

```suggestion
    func jan(_ day: Int) { DateComponents(calendar: .current, timeZone: .current, year: 2019, month: 1, day: day).date! }
```

(and yes, I know those dates came from my code :( )

Reply:
Good idea, I like it.